### PR TITLE
Activity model with Activities list in Scenario

### PR DESCRIPTION
**Pull request by paulinaborys**

### CODE REVIEW

File: grisera_api/activity/activity_model.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    """
+    The type of activity
+
+    Attributes:
+        individual (str): Individual activity
+        two_people (str): Two people activity
+        group (str): Group activity
+    """
+    individual = "individual"
+    two_people = "two-people"
```
Comment (mrwrob) [inline grisera_api/activity/activity_model.py:18]: Should we use `-` in enums' strings? @RioluM, in `observable_information_model.py` we've got spaces. I think we should be consistent. WDYT?

File: grisera_api/scenario/scenario_service.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            scenario (ScenarioIn): Scenario to be added
+
+        Returns:
+            Result of request as scenario object
+        """
+
+        if scenario.activities is not None:
+            for activity in scenario.activities:
+                # Create Nodes Activity for scenario
+                node_response_activity=self.activity_service.save_activity(activity=activity)
```
Comment (mrwrob) [inline grisera_api/scenario/scenario_service.py:30]: Are we using this `node_response_activity`? If not, I think we could just call the `save_activity` method without assigning the response to the variable.

**Review by mrwrob — COMMENTED** [review]
What about tests? Don't we need them here?

File: grisera_api/activity/activity_model.py (hunk 1 above)
Comment (RioluM) [reply grisera_api/activity/activity_model.py:18]: Oh, indeed it's used as a one word. In that case i would leave it with -. What do you think @mrwrob ?

**Comment by paulinaborys** [comment]
I have just added tests for activity.

**Review by RioluM — APPROVED** [review]

**Pull request merged**